### ISSUE TITLE
Master cannot be built due to an error in component gameViewPixi

### ISSUE THREAD

**Issue by OskarKlintrot** [OskarKlintrot/1dv411-project#33]
Travis can't find the module `ReactPixi`. I don't think `ReactPixi` should live in the `scripts` folder since that is where scripts that can't be boundled by webpack lives. I'm not sure but I think that `./` might be causing the issue.

> ```ERROR in ./src/app/components/gameViewPixi/index.jsx
> Module not found: Error: Cannot resolve 'file' or 'directory' ./../../scripts/ReactPixi in /home/travis/build/OskarKlintrot/1dv411-project/src/app/components/gameViewPixi
>  @ ./src/app/components/gameViewPixi/index.jsx 15:17-53```

https://travis-ci.org/OskarKlintrot/1dv411-project#L780-L782

**Comment by johnnypesola** [comment]
Moved ReactPixi  related files to `src/app/components/gameViewPixi/lib/`. in the branch `game-view-pixi`

Made sure `game-view-pixi` branch reflected the `master` branch. And I then restored the `master` branch to how it was before `game-view-pixi` merge.

The `game-view-pixi` branch now is a reflection of the `master` branch with a game view and pixi.js implemented in it.

The `dev` branch should merge with `master` in the future without any conflicts.

The `game-view-pixi` branch will not be kept up to date with dev or master from this point on. It only exists for historical purposes or as an emergency backup branch.
